pkg/cluster/compute/affinityrule/affinityrule.go

```diff
@@ -28,21 +38,30 @@ func NewAffinityRule(instance *wmi.WmiInstance) (*AffinityRule, error) {
 	return &AffinityRule{wmiafRule}, nil
 }
 
-// CreateAffinityRule
-func CreateAffinityRule(whost *host.WmiHost, name string, ruleType int) (affinityRule *AffinityRule, err error) {
+// Caller should call Close() once done using this instance
+func GetAffinityRuleClass(whost *host.WmiHost) (*wmi.WmiClass, error) {
```

Comment:
follow lower casing for private methods?

Reply:
Fixed in new patch